clouddriver-google/src/main/groovy/com/netflix/spinnaker/clouddriver/google/controllers/GoogleLoadBalancerController.groovy

```diff
@@ -69,8 +67,8 @@ class GoogleLoadBalancerController {
             account: view.account,
             region: view.region,
             name: view.name,
-            loadBalancerType: loadBalancerType,
-            backendServices: loadBalancerType == HTTP ? backendServices.unique() as List<String> : null
+            loadBalancerType: loadBalancerType as String,
```

Comment:
Any reason not to also make `GoogleLoadBalancerSummary.loadBalancerType` the `enum` type instead of a `String`?